### DISCUSSION TITLE
Can't add configured features to biome gen

### DISCUSSION THREAD

**Discussion by janMitame** [misode/misode.github.io#321] [Q&A]
I've got my features configured, but I can't add them to the list of placed features. I tried manually typing them in but when I load the pack it gives the error "Datapack Verificaton Failed". Anyone know how to fix this? 

edit: I tried removing the list and just not having any features and it still isn't working. aaa
edit2: anything with the fairy namespace isn't working. I can't even add the fairy:fairy-forest biome to the dimension

[Uploading Land_of_the_Fairies.zip…]()

**Comment by misode** [comment]
You might get faster help on the Minecraft Configs discord server: https://discord.gg/JfxHv6KpaH
